src/main/java/mate/academy/Main.java

```diff
@@ -39,17 +53,43 @@ public static void main(String[] args) {
         tomorrowMovieSession.setMovie(fastAndFurious);
         tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(1L));
 
-        MovieSession yesterdayMovieSession = new MovieSession();
-        yesterdayMovieSession.setCinemaHall(firstCinemaHall);
-        yesterdayMovieSession.setMovie(fastAndFurious);
-        yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
+        MovieSession nextMovieSession = new MovieSession();
+        nextMovieSession.setCinemaHall(firstCinemaHall);
+        nextMovieSession.setMovie(fastAndFurious);
+        nextMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
```

Comment:
The `nextMovieSession` is set with a showtime in the past (`LocalDateTime.now().minusDays(1L)`). Ensure this is intentional, as it might cause issues when adding sessions to the shopping cart or finding available sessions.